tests/testconfig/metrics.yaml

```diff
@@ -133,3 +133,141 @@ pad_full_balance:
     same_result_only_train: -1.75
     diff_result: -1.5
     diff_result_only_train: -1.3684210526315788
+kde_umap_kl_approx:
+  class: kde
+  arguments:
+    embedding_name: inception_umap
+    embedding_kwargs:
+      n_components: 2
+    kernel_name: gaussian
```

Comment:
I think this branch may have need based on an earlier version of develop, as this file now conflicts with the one in main. The only real difference is everything should be nested under `metrics` for consistency with those we use for the experiments. Develop will need to be merged in before this PR gets merged into develop

Reply:
have merged in develop